modules/webauthorprofile/lib/webauthorprofile_webinterface.py

```diff
@@ -398,13 +398,19 @@ def index(self, req, form):
         profile_page = WebProfilePage("profile", webapi.get_longest_name_from_pid(self.person_id))
         profile_page.add_profile_menu(menu)
 
+        if 'form_email' in pinfo:
+            gFormEmail = pinfo['form_email']
```

Comment:
If the user is not guest, and hence the email is already available in the user session, will it be correctly available as usual?